src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      class="search_bar"
+      data-qa="big"
+    >
+      <label class="search_label">
+        <input
+          class="bar_inpute big_inpute"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+        <img
+          class="imageBig"
```

Comment:
To get clear code, you should decide what type of cases use all over the project for naming classes. 
Here you have used as kebab-case, as camelCase.
So, please rewrite all classes using ONE case